DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni

```diff
@@ -184,6 +184,11 @@
 #string STR_MEMORY_PROFILE_XMP2    #language en-US  "XMP#2 (predefined extreme memory profile)"
 #string STR_MEMORY_PROFILE_XMP3    #language en-US  "XMP#3 (predefined extreme memory profile)"
 
+#string STR_MEMORY_IBECC_PROMPT   #language en-US  "Memory In-Band ECC (IBECC) Enable"
+#string STR_MEMORY_IBECC_HELP     #language en-US  "This option enables In-Band ECC (IBECC) feature on Intel processors\n\n"
+                                                   "Firmware will reserve a portion of RAM to store ECC syndrome data and enable hardware support for error correction on whole RAM.\n\n"
+                                                   "The portion of RAM being reserved depends on total RAM size."

```

Comment:
maybe mention what percentage of ram will be consumed? I'm asssuming that's a static value

Reply:
https://github.com/Dasharo/edk2/compare/d0380f96055ef30498201cee219ab21b2d284f60..dbb61a9b40c091128ffb3faf66b16521915aa253